avm/res/storage/storage-account/main.bicep

```diff
@@ -653,31 +661,31 @@ module secretsExport 'modules/keyVaultExport.bicep' = if (secretsExportConfigura
       contains(secretsExportConfiguration!, 'accessKey1')
         ? [
             {
-              name: secretsExportConfiguration!.accessKey1
+              name: secretsExportConfiguration!.?accessKey1
               value: storageAccount.listKeys().keys[0].value
             }
           ]
         : [],
       contains(secretsExportConfiguration!, 'connectionString1')
         ? [
             {
-              name: secretsExportConfiguration!.connectionString1
+              name: secretsExportConfiguration!.?connectionString1
               value: 'DefaultEndpointsProtocol=https;AccountName=${storageAccount.name};AccountKey=${storageAccount.listKeys().keys[0].value};EndpointSuffix=${environment().suffixes.storage}'
             }
           ]
         : [],
       contains(secretsExportConfiguration!, 'accessKey2')
         ? [
             {
-              name: secretsExportConfiguration!.accessKey2
+              name: secretsExportConfiguration!.?accessKey2
               value: storageAccount.listKeys().keys[1].value
             }
           ]
         : [],
       contains(secretsExportConfiguration!, 'connectionString2')
         ? [
             {
-              name: secretsExportConfiguration!.connectionString2
+              name: secretsExportConfiguration!.?connectionString2
```

Comment:
Maybe not the time and place (but I'll leave you to be the judge of that): The AVM spec that each secret name should have the naming convention `<secret>Name` - so e.g. this property's name should not be `connectionString2`, but `connectionString2Name`.